lib/tokens.js

```diff
@@ -406,19 +412,23 @@ export async function resolve({requester, token, levelOfAssurance}) {
         });
         if(bs.get(index)) {
           // token resolved for same requester, return pairwise token
-          const record = await _getPairwiseToken({internalId, requester});
-          const {pairwiseToken: {value: pairwiseToken}} = record;
-          return {pairwiseToken};
+          tokenRecord = await _getPairwiseToken({internalId, requester});
+          if(allowResolvedInvalidatedTokens) {
+            const {pairwiseToken: {value: pairwiseToken}} = tokenRecord;
+            return {pairwiseToken, internalId};
+          }
         }
       }
+      if(!tokenRecord) {
       // token already resolved to another requester, can only be
       // scope-resolved once
```

Comment:
Maybe github isn't showing this diff to me properly, but this comment should be indented now to match the conditional... if actually isn't.

```suggestion
        // token already resolved to another requester, can only be
        // scope-resolved once
```

Reply:
The comment is in line with `if(!tokenRecord) {`